lib/video_player_web_hls.dart

```diff
@@ -163,41 +164,40 @@ class _VideoPlayer {
     // ignore: undefined_prefixed_name
     ui.platformViewRegistry.registerViewFactory(
         'videoPlayer-$textureId', (int viewId) => videoElement);
-    if(uri.toString().contains("m3u8"))
-    {
+    if (isSupported() && uri.toString().contains("m3u8")) {
```

Comment:
I added this isSupported conditional as well.  I'm not sure it's actually necessary for this particular bug, but it seems to be the conditional that hls.js examples rely on.